SPEC.md

```diff
@@ -5816,13 +5842,13 @@ task test_runtime_info_task {
   
   output {
     Boolean at_least_two_gb = task.memory >= (2 * 1024 * 1024 * 1024)
-    Int return_code = task.return_code
+    Int return_code = select_first([task.return_code, 0])
```

Comment:
The spec states that the value is only guaranteed to be available in the `output` section. It doesn't make clear whether that means it's now an `Int` in that context or just an `Int?`. I guess I agree that it makes the most sense reading the spec to interpret it as an `Int?` everywhere, but it seems confusing.

At a minimum, I think I would suggest two changes here:

- The idea that we are `select_first`-ing this with 0 seems... strange. Perhaps we can just make the output an `Int?`.
- We should file an issue to further define whether this value is an `Int?` in the output section or if it can just be an `Int`.